module/scratchpad.lua

```diff
@@ -71,7 +105,39 @@ function Scratchpad:turn_off()
     local c = matches[1]
     if c then
         c.sticky = false
-        helpers.client.turn_off(c)
+
+        -- Get the tweens
+        local anim_x = self.awestore.x
+        local anim_y = self.awestore.y
+
+        -- Subscribe
+        if anim_x then
+            anim_x:subscribe(function(x) if c then c.x = x end end)
+        end
+        if anim_y then
+            anim_y:subscribe(function(y) if c then c.y = y end end)
+        end
+
+        -- Unsubscribe
+        if anim_x then
+            anim_x:set(anim_x:initial())
+            local unsub
+            unsub = anim_x.ended:subscribe(
+                        function()
+                    helpers.client.turn_off(c)
+                    unsub()
+                end)
+        end
+        if anim_y then
+            anim_y:set(anim_y:initial())
+
+            local unsub
+            unsub = anim_y.ended:subscribe(
+                        function()
+                    helpers.client.turn_off(c)
+                    unsub()
+                end)
+        end
```

Comment:
The client won't get turned off at all if neither `anim_y` nor `anim_x` are set. Add something like `if not anim_x and not anim_y then helpers.client.turn_off(c) end` at the end of the `turn_off` function to fix that